docs/resources/control-flow/user-interactivity/forms/form-widgets/checkbox.md

```diff
@@ -0,0 +1,527 @@
+
+![Circular checkbox](../../imgs/make-checkbox-circular.avif)
+
+#### Disable Checkbox
+
+You may need to disable a checkbox if certain conditions aren't met. For instance, users should only be able to use the 'Same as Shipping Address' checkbox when a shipping address is provided.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://www.loom.com/embed/8c69f44814ff492f81c85ddb670beb73?sid=361442f7-9d26-4fb0-a4aa-86d9fece59db"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+To disable a checkbox, move to the **Properties Panel** **>** turn on the **Checkbox Disable Options >** click **Unset,** and set the [**Condition**](../../../../control-flow/functions/conditional-logic.md). Once set, you could also customize the disabled state colors using the *Disabled Check Color* property.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://www.loom.com/embed/fcacb70c575541a08dceef5487ff9a8a?sid=824f6560-e8e1-488a-86e4-1ee6e9a07465"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+## CheckboxListTile
+
+The **CheckboxListTile** widget combines the functionality of a checkbox with a [ListTile](../../../../ui/widgets/built-in-widgets/list-grid.md#listtile-widget), providing a more comprehensive option for displaying checkboxes alongside additional information. Unlike the Checkbox this widget includes a title, and an optional subtitle, all within a single, cohesive element.
+
+CheckboxListTile is ideal for use cases where you want to provide more context or descriptive text alongside the checkbox, such as in a settings menu or a form with detailed options.
+
+
+## CheckboxGroup
+
+The **CheckboxGroup** widget allows you to present a group of checkboxes as a single entity. This is particularly useful when you want users to select multiple options from a list. Each checkbox within the group can be checked or unchecked independently of the others.
+
+:::tip[Widget State]
```

Comment:
This should be added to the top of page as all the widgets here have a widget state. Add them before talking about the variations in details.

Reply:
Thanks for catching!